src/translations/es.json

```
@@ -434,5 +434,8 @@
     "pointsAndWins": "Puntos y victorias",
     "pointsOnly": "Solo puntos",
     "winsOnly": "Solo gana"
+  },
+  "error": {
+    "title": "404 No Encontrado"
```

Since 404 is a specific error, I think it makes sense to nest this under a "404" key like:
```suggestion
  "error": {
    "404": {
      "title": "404 No Encontrado"
    }
  }
```

That way we can have whole error pages for other errors and they can have separate titles etc